src/library_workerfs.js

```diff
@@ -0,0 +1,114 @@
+mergeInto(LibraryManager.library, {
+  $WORKERFS__deps: ['$FS'],
+  $WORKERFS: {
+    DIR_MODE: {{{ cDefine('S_IFDIR') }}} | 511 /* 0777 */,
+    FILE_MODE: {{{ cDefine('S_IFREG') }}} | 511 /* 0777 */,
+    reader: null,
+    mount: function (mount) {
+      assert(ENVIRONMENT_IS_WORKER);
+      WORKERFS.reader = new FileReaderSync();
```

Comment:
multiple calls to `mount` will unnecessarily create the reader, can do `if (!WORKERFS.read) WORKERFS.reader = ..`